src/main/java/net/sf/jabref/logic/formatter/bibtexfields/HtmlToUnicodeFormatter.java

```diff
@@ -0,0 +1,36 @@
+package net.sf.jabref.logic.formatter.bibtexfields;
+
+import net.sf.jabref.logic.formatter.Formatter;
+import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.layout.LayoutFormatter;
+
+import org.apache.commons.lang3.StringEscapeUtils;
+
+public class HtmlToUnicodeFormatter implements LayoutFormatter, Formatter {
```

Comment:
I meant http://help.jabref.org/en/CustomExports under `Built-in export formatters` since the class is implementing LayoutFormatter and thus can be used for the exports too, right?